### ISSUE TITLE
Poor dmesg usage in tempesta.sh

### ISSUE THREAD

**Issue by EvgeniiMekhanik** [tempesta-tech/tempesta#2199]
When we start tempesta we check that "Tempesta FW is ready" is present in dmesg. But if there are a lot of messages in dmesg (if access_log is on for example) and dmesg buffer is not big, this phrase is rewritten by logs. So Tempesta FW is not start. It is a problem when Tempesta is a service which should automatically start after reboot, but can't do it.

UPDATE:
It seems that we should not check dmesg at all. We should check only Tempesta state and do not unload modules if Tempesta FW fails live reconfiguration

**Comment by krizhanovsky** [comment]
Can we just increase dmesg buffer in the script?

**Comment by krizhanovsky** [comment]
I think the statement
```
        # If 'net.tempesta.state' entry exists but Tempesta start process has
        # been failed for some reason (e.g. configuration parsing error), then
        # 'sysctl' does not indicate any problems and exits with zero code;
        # so, stderr may return 'sysctl: setting key "net.tempesta.state"'
        # followed by some error message or just empty string.
        # The most reliable way to check Tempesta status is to check
        # net.tempesta.state and if it is "start" check dmesg.
```
is wrong: the file `/proc/tempesta/state` must provide sufficient and reliable way for `tempesta.sh` to determine the state of Tempesta FW. If we fail to reconfigure, we can extend the set of available values in the file. We should get rid of dmesg parsing and fix `/proc/tempesta/state`

**Comment by krizhanovsky** [comment]
Fixed in #2200
